packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -132,6 +132,21 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
   };
 
   const seoComponent = <NextSeo {...seo} />;
+  const schemaComponent = (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{
+        __html: JSON.stringify({
+          '@context': 'https://schema.org',
+          '@type': 'Article',
+          headline: post?.title,
+          datePublished: post?.createdAt,
+          dateModified: post?.updatedAt,
+          description: getSeoDescription(post),
+        }),
+      }}
```

Comment:
should we sanitize these? -- since this is user data, it can expose us to attacks

this might be a non-issue since the JSON-LD schema is just json, not executable JS, but still worth double checking

Reply:
@kopancek worth checking what happens if stringify raises an error

Reply:
I just checked. If `JSON.stringify()` fails and throws an error in a script tag in the head ofthe HTML document, it will not "break" the page in the sense of causing a blank screen or a browser crash. It will just stop execution of the current script that has the SEO JSON. So in the worst case, we are not sending that piece to the search engine.